features/ubuntu_pro_fips.feature

```diff
@@ -0,0 +1,150 @@
+Feature: Command behaviour when auto-attached in an ubuntu PRO fips image
+
+    @series.lts
+    @uses.config.machine_type.aws.pro.fips
+    Scenario Outline: Proxy auto-attach in an Ubuntu pro fips AWS machine
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I launch a `focal` `proxy` machine
+        And I run `apt install squid -y` `with sudo` on the `proxy` machine
+        And I add this text on `/etc/squid/squid.conf` on `proxy` above `http_access deny all`:
+            """
+            dns_v4_first on\nacl all src 0.0.0.0\/0\nhttp_access allow all
+            """
+        And I run `systemctl restart squid.service` `with sudo` on the `proxy` machine
+        When I create the file `/etc/ubuntu-advantage/uaclient.conf` with the following:
+        """
+        contract_url: 'https://contracts.canonical.com'
+        data_dir: /var/lib/ubuntu-advantage
+        log_level: debug
+        log_file: /var/log/ubuntu-advantage.log
+        ua_config:
+          http_proxy: http://<ci-proxy-ip>:3128
+          https_proxy: http://<ci-proxy-ip>:3128
+        """
+        And I verify `/var/log/squid/access.log` is empty on `proxy` machine
+        When I run `ua auto-attach` with sudo
+        Then stdout matches regexp:
+            """
+            .*esm-apps      +yes +enabled +UA Apps: Extended Security Maintenance \(ESM\)
+            esm-infra     +yes +enabled +UA Infra: Extended Security Maintenance \(ESM\)
+            fips          +yes +<fips-s> +NIST-certified core packages
+            fips-updates  +yes +disabled +NIST-certified core packages with priority security updates
+            livepatch     +yes +n/a  +Canonical Livepatch service.*
+            """
+        When I run `cat /var/log/squid/access.log` `with sudo` on the `proxy` machine
+        Then stdout matches regexp:
+        """
+        .*CONNECT contracts.canonical.com.*
+        """
+        And stdout does not match regexp:
+        """
+        .*CONNECT 169.254.169.254.*
+        """
+        Examples: ubuntu release
+           | release | fips-s   |
+           | xenial  | enabled  |
+           | bionic  | enabled  |
+
+    @series.lts
+    @uses.config.machine_type.aws.pro.fips
+    Scenario Outline: Check fips is enabled correctly on Ubuntu pro fips AWS machine
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I create the file `/etc/ubuntu-advantage/uaclient.conf` with the following:
+        """
+        contract_url: 'https://contracts.canonical.com'
+        data_dir: /var/lib/ubuntu-advantage
+        log_level: debug
+        log_file: /var/log/ubuntu-advantage.log
+        """
+        And I run `ua auto-attach` with sudo
+        And I run `ua status --wait` as non-root
+        And I run `ua status` as non-root
+        Then stdout matches regexp:
+            """
+            .*esm-apps      +yes +enabled +UA Apps: Extended Security Maintenance \(ESM\)
```

Comment:
Do we need the `.*` here?
```suggestion
            esm-apps      +yes +enabled +UA Apps: Extended Security Maintenance \(ESM\)
```

Reply:
Not really, I had made changes when I testing some parts, and forgot to delete, my bad.